src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Updated parameter -BlobType description in help file of Set-AzStorageBlobContent
```

Comment:
Change log is not required in a doc-only pull request. We push every doc of every module to docs.microsoft.com in every release :)